rpki-validator/src/main/resources/application.properties

```diff
@@ -92,3 +92,8 @@ management.endpoints.enabled-by-default=false
 management.endpoint.prometheus.enabled=true
 management.endpoints.web.exposure.include=prometheus
 
+# Enabling strict validation will not allow stale manifest, stale crl, and reject manifest with non existing entry.
+rpki.validator.strict-validation=false
+
+# Enabling rsync only will ignore RRDP repositories and only use rsync
```

Comment:
Slightly too wordy. 
"This disables RRDP, so the validator only uses rsync to download repositories."?